core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -31,15 +31,20 @@ THE SOFTWARE.
   <st:include page="sidepanel.jelly" />
   <f:breadcrumb-config-outline />
   <l:main-panel>
+    <div class="jenkins-app-bar">
+      <div class="jenkins-app-bar__content">
+        <h1>${%Configure System}</h1>
+      </div>
+    </div>
     <div class="behavior-loading"><l:spinner text="${%LOADING}"/></div>
     <f:form method="post" name="config" action="configSubmit">
       <j:set var="instance" value="${it}" />
       <j:set var="descriptor" value="${instance.descriptor}" />
 
-      <f:entry title="${%Home directory}" help="/help/system-config/homeDirectory.html">
-        ${it.rootDir}
+      <f:entry title="${%Home directory}" description="By default, Jenkins stores all of its data in this directory on the file system" help="/help/system-config/homeDirectory.html">
+        <div class="jenkins-quote jenkins-quote--monospace">${it.rootDir}</div>
       </f:entry>
-      <f:entry title="${%System Message}" help="/help/system-config/systemMessage.html">
+      <f:entry title="${%System Message}" description="This message will be displayed at the top of the Jenkins main page. This can be useful for posting notifications to your users">
```

Comment:
```suggestion
      <f:entry title="${%System Message}" description="${%This message will be displayed at the top of the Jenkins main page. This can be useful for posting notifications to your users}">
```